std/cxx/std/Array.hx

```diff
@@ -17,4 +17,12 @@ extern class Array<T, @:const Count: Int>  {
 	public function maxSize(): cxx.num.SizeT;
 
 	public function fill(value: T): Void;
+
+	@:arrayAccess
+        @:nativeFunctionCode("((*{this})[{arg0}])")
```

Comment:
I propose to map this to `at`: https://en.cppreference.com/w/cpp/container/array/at

Reply:
You've got a good point.

Reply:
However, I sometimes don't want to waste time on bound checking if I don't need to.